packages/git/src/browser/git-view-contribution.ts

```diff
@@ -29,6 +29,10 @@ export const GIT_WIDGET_FACTORY_ID = 'git';
 export const EDITOR_CONTEXT_MENU_GIT = [...EDITOR_CONTEXT_MENU, '3_git'];
 
 export namespace GIT_COMMANDS {
+    export const CLONE = {
+        id: 'git.clone',
```

Comment:
Please add the missing `readonly`s.

Reply:
sorry not sure which `readonly` you are talking about :/